Carbon/navigation/screens/Questionnaire/QuestionMileage.js

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import {View, Text,Button,TextInput } from 'react-native';
+import { Colors } from '../../../colors/Colors';
+
+/*
+Mileage Screen
+
+TODO: Improve UI
+TODO: Improve transferring of data between pages
+*/
+
+export default function MileageScreen({navigation,route}) {
+    //Values from previous pages
+    const dietScore = route.params?.dietScore;
+    const homePowerScore = route.params?.homePowerScore;
+    const annualPower = route.params?.annualPower;
+    const transportScore= route.params?.transportScore;
+
+    //Value to calculate & transfer at the "finished" screen
+    const [miles,setMiles] = useState(0);
+
+    return (
+    <>
+    <View
+    style={{
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: "100%"
+    }}
+    >
+        <Text>
+        (Optional) What is the mileage on your vehicle?
```

Comment:
Approving now, but can we change this to miles per gallon on your vehicle or how many miles do you drive per week? Like I mentioned previously this question is tough since most people by used cars. For example, my car I got with 107k miles on it - those miles shouldn't count against me.

Reply:
Noted! I'll add a commit right now to include in the PR